doc/tools/apigen.py

```diff
@@ -207,7 +207,11 @@ def _parse_module_with_import(self, uri):
         classes : list of str
             A list of (public) class names in the module.
         """
-        mod = __import__(uri, fromlist=[uri])
+        try:
+            mod = __import__(uri, fromlist=[uri])
+        except ImportError:
+            # fix ImportError for submodules in Python 3
+            mod = __import__(uri, fromlist=[])
```

Comment:
`fromlist=['']` works fine on Python 3 for me, but with Python 2.7 I get the following error:
```
Traceback (most recent call last):
  File "tools/build_modref_templates.py", line 73, in <module>
    docwriter.write_api_docs(outdir)
  File "/media/Data1/src_repositories/my_git/nibabel_grl/doc/tools/apigen.py", line 470, in write_api_docs
    self.write_modules_api(modules, outdir)
  File "/media/Data1/src_repositories/my_git/nibabel_grl/doc/tools/apigen.py", line 434, in write_modules_api
    head, body = self.generate_api_doc(m)
  File "/media/Data1/src_repositories/my_git/nibabel_grl/doc/tools/apigen.py", line 273, in generate_api_doc
    functions, classes = self._parse_module_with_import(uri)
  File "/media/Data1/src_repositories/my_git/nibabel_grl/doc/tools/apigen.py", line 210, in _parse_module_with_import
    mod = __import__(uri, fromlist=[''])
  File "/home/lee8rx/anaconda/lib/python2.7/site-packages/nibabel/freesurfer/__init__.py", line 4, in <module>
    from .io import read_geometry, read_morph_data, write_morph_data, \
  File "/home/lee8rx/anaconda/lib/python2.7/site-packages/nibabel/freesurfer/io.py", line 8, in <module>
    from .. externals.six.moves import xrange
ImportError: No module named externals.six.moves
```